### PR TITLE
Better Error for "Invalid color"

### PR DESCRIPTION
**Pull request by Wilma456**
the "Invalid color" error now contain the wrong color.

### CODE REVIEW

**Comment by SquidDev** [comment]
Would it also be a good idea to include the value in the `TermAPI` error messages? I realise that is an out of bounds error instead, but might be a little more consistent.

File: src/main/resources/assets/computercraft/lua/rom/apis/window.lua [hunk 1]
```diff
@@ -321,7 +321,7 @@ function create( parent, nX, nY, nWidth, nHeight, bStartVisible )
         if type( color ) ~= "number" then
             error( "bad argument #1 (expected number, got " .. type( color ) .. ")", 2 )
         elseif tHex[color] == nil then
-            error( "Invalid color", 3 )
+            error( "Invalid color (got " .. color .. ")", 2 )
```
Comment (lupus590) [inline src/main/resources/assets/computercraft/lua/rom/apis/window.lua]: the 2 was a 3, is that an intended change?

Comment (Wilma456) [inline src/main/resources/assets/computercraft/lua/rom/apis/window.lua]: Yes. There must be a 2 and not a 3.

Comment (SquidDev) [inline src/main/resources/assets/computercraft/lua/rom/apis/window.lua]: @lupus590 I presume the 3 was a relic from when `window.setBackgroundColor` and `window.setBackgroundColour` were separate functions which deligated to this.

**Pull request merged**